pkg/localize/locales/en/cmd/status.en.toml

```diff
@@ -6,7 +6,7 @@ one = '''
 View status information of your currently used services.
 Choose to view the status of all services with "rhoas status" or specific services with "rhoas status [service]"
 
-To use a different service run "rhoas <service> use [args] [flags]".
+To use a different service run "rhoas service use [args] [flags]".
```

Comment:
Not sure how to properly solve this, there is no rhoas service command

Reply:
Thought - could we find a replacement for "<>" which has the same effect, like "[]"?

```shell
rhoas [service] use
```

Reply:
+100 to use `[]` if we use them can we replace all `<>` with `[]` as part of the PR and not make any other changes.